src/components/Candidates/CandidatesJoinUs/CandidatesOpenPositions/FormsModal/FormsModal.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import { Form, FormGroup, Label,Col, Input,Row, Modal, ModalHeader, ModalBody, Button} from 'reactstrap';
+import './FormsModal.css';
+import emailjs from 'emailjs-com';
+
+const service_id = 'juarezid_1234_4321_01';
+const template_id = 'template_6owvhzc';
+const user_id = 'user_exlhFGfiCpajm8iHQostc';
+
+class FormsModal extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.createMail = this.createMail.bind(this);
+  }
+
+  createMail(){
+    const mes = "Hola soy " + this.name.value + " estudiante de " + this.carrera.value + " de " + this.semester.value + " semestre. \n" + this.comments.value;
+    return({
+      message: mes, 
+      from_name: this.name.value, 
+      reply_to: this.matricualtionNumber.value + '@itesm.mx',
+      puesto : this.props.selectedPosition.title,
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.toggle();
+    console.log(this.name.value + " " + this.matricualtionNumber.value + " " + this.semester.value + " " + this.props.selectedPosition.title + " " + this.comments.value);
+    alert("Thanks for your interest " + this.name.value + "! \nCheck your tec email");
+
+    emailjs.send(service_id,template_id , this.createMail(), user_id)
+    .then((result) => {
+        console.log(result.text);
+    }, (error) => {
+        console.log(error.text);
+    });  
+  }
+
+  render() {
+
+    const greeting = "Join us as " + this.props.selectedPosition.title + "!";
```

Comment:
For small strings, I suggest you to use text chains to make the code smaller
```suggestion
    const greeting = `Join us as ${this.props.selectedPosition.title}!`;
```